test/examples/positionGlobe.js

```diff
@@ -37,7 +37,7 @@ describe('positionGlobe', () => {
                 globeView.notifyChange(true);
             }));
 
-        await page.screenshot({ path: '/tmp/b.png' });
+        await page.screenshot({ path: `${pathToTemp}/b.png` });
```

Comment:
[os.tmpdir()](https://nodejs.org/api/os.html#os_os_tmpdir) is a better idea than `global.pathToTemp` IMHO.

Reply:
yes

Reply:
done